CONTRIBUTING.md

```diff
@@ -179,3 +179,43 @@ Do your best to follow the clean code guidelines.
 - Don’t return an error code, throw an exception instead.
 
 Ref: Book: Clean Code by Robert C. Martin (Uncle Bob)
+
+## Compilation and Installation
+
+This guide is helpful to compile and install Nmstate from source, for installing stable release or other installation methods, pleaser refer to [Nmstate installation guide](https://nmstate.io/user/install.md)
+
+### Clone the source
+```bash
+git clone https://github.com/nmstate/nmstate.git
+cd nmstate
+```
+
+#### Install build tool
+Install cargo using rustup (recommended): https://sh.rustup.rs.
```

Comment:
This link is not to the installation instructions. Better link to the instructions.